### ISSUE TITLE
MyFavorites Accessibility issues

### ISSUE THREAD

**Issue by SNefertitiB** [gcivil-nyu-org/team-1-inperson#207]
- VoiceOver doesn't read out "My Favorites"
- VoiceOver only tells that search bar searches by address
- VoiceOver doesn't read any of the details on the cards. Should say something about the map and the address on the card.
